src/components/app-header/app-header.styles.js

```diff
@@ -1,8 +1,10 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { changeCurrency } from '../../redux/currency/currency.actions';
```

Comment:
For what you need those import here?

Reply:
Its my mistake,I have allready remove this unusable import